shelter/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/shelter
```

Comment:
It's better to change `module github.com/shelter` to `module github.com/inclavare-containers/shelter` which is same as `rune`、`epm` and `sgx-tools`.